.github/CODEOWNERS

```diff
@@ -7,8 +7,8 @@
 # Applications
 /packages/manager/ @ovh/team-manager-dev
 
-# Super-components
-/packages/super-components @ovh/team-manager-transfo @ovh/su-digital-tools-core-stack
+# Manager-components
+/packages/manager-components @ovh/team-manager-transfo
```

Comment:
**question:** What's about having a `components` folder into `packages/manager/` instead?

Reply:
We had a lot of discussions with Anoop and core team few weeks ago concerning the folder name, we have change like 3 times.